OpenRA.Mods.Common/Traits/Player/HarvesterAttackNotifier.cs

```diff
@@ -47,12 +47,16 @@ public HarvesterAttackNotifier(Actor self, HarvesterAttackNotifierInfo info)
 
 		public void Damaged(Actor self, AttackInfo e)
 		{
-			// only track last hit against our harvesters
-			if (!self.Info.HasTraitInfo<HarvesterInfo>())
+			// Don't track self-damage
+			if (e.Attacker != null && e.Attacker.Owner == self.Owner)
 				return;
 
-			// don't track self-damage
-			if (e.Attacker != null && e.Attacker.Owner == self.Owner)
+			// This player won't hear us anyway
+			if (!self.Owner.Playable)
```

Comment:
I think I'll just drop this, there are no real benefits and this just causes such trouble.